### DISCUSSION TITLE
哪个客户端支持多端口（端口跳跃）呀？

### DISCUSSION THREAD

**Discussion by wind0ta** [apernet/hysteria#511] [Q&A]
手机上小火箭只能设置1个端口，windows的nekoRay貌似也只能1个端口。

**Comment by hang333** [comment]
v2rayN

**Reply by taohuajianxian** [reply]
能否细说怎么设置呢？

**Comment by kilvn** [comment]
手机上小火箭一周前更新了已经支持端口跳跃了，Mac上面用drawin客户端端口跳跃后开socks5给clashx，我是这么做的。

**Reply by taohuajianxian** [reply]
支持，但极不稳定，非常非常不稳定，谜一样的连通性。

**Comment by xiaorong61** [comment]
nekobox for android

**Reply by taohuajianxian** [reply]
唯一实测正解

**Reply by z80sui** [reply]
怎么设置呀 请教

**Reply by taohuajianxian** [reply]
服务器端请参考官网写的非常详细，就是端口转发而已。客户端端口号那里填写一个范围比如10000-20000这样的格式就可以了。

**Reply by xiaorong61** [reply]
> 怎么设置呀 请教

摸索一下就会了

**Reply by ohmycbc** [reply]
我也用nekobox for android。。。没用，只能写一个端口

**Reply by Hcc422** [reply]
我也用nekobox 确实能跳跃的，说不能的是软件输入不了还是什么？建议检查一下有没有配置好iptable（不写入开机自动每次重启都会失效），防火墙有没有放行
